mirdata/jams_utils.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""functions for converting mirdata annotations to jams format
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import jams
+
+def jams_converter(
+        beat_data=None,
+        chord_data=None,
+        note_data=None,
+        f0_data=None,
+        section_data=None,
+        key_data=None,
+        artist=None,
+        duration=None,
+        title=None,
+        annotator=None):
+
+    jam = jams.JAMS()
+
+    # metadata
+    if artist is not None:
+        jam.file_metadata.artist = artist
+    if title is not None:
+        jam.file_metadata.title = title
+    if duration is not None:
+        jam.file_metadata.duration = duration
+
+    # annotations
+
+    # beats
```

Comment:
let's break each of these into separate functinos - input is a list, output is a jams annotation object